### ISSUE TITLE
Duplicate element ids for pane tabs

### ISSUE THREAD

**Issue by gtritchie** [rstudio/rstudio#5011]
### System details

    RStudio Edition : Desktop or Server
    RStudio Version : 1.3.274
    OS Version      : any
    R Version       : any

### Steps to reproduce the problem
- load the IDE
- run an automated accessibility test against the page (Lighthouse or axe) and/or examine the HTML directly in dev tools

### Describe the problem in detail
- each tab pane (history, console, environment, etc.) should have a unique ID in the HTML such as `id="rstudio_workbench_tab_environment"` and so on; however, in many cases, the same ID is found twice (looks related to minimized tab variations)

### Describe the behavior you expected
- no duplicate IDs; having duplicates is a WCAG 2.0 and 2.1 level A MUST fix
- note that there are other instances of duplicate IDs; this issue is specifically to track the ones related to normal and minimized pane tabs
